src/App/Application.cpp

```diff
@@ -171,11 +171,6 @@
 FC_LOG_LEVEL_INIT("App", true, true)
 
 using namespace App;
-using namespace Base;
-using namespace std;
-using namespace boost;
-using namespace boost::program_options;
-using Base::FileInfo;

```

Comment:
Not all cases of `using` is bad, and `using Base::FileInfo;` is an example of one that is ok to keep.